package.json

```diff
@@ -25,6 +25,7 @@
     "start": "npm run build && nodemon ./src/index.js",
     "start:debug": "node --inspect-brk=0.0.0.0 ./src/index.js",
     "start:docker": "docker-compose -f docker-compose.yaml -f docker-compose.override.yaml up --build",
+    "start:docker:local": "docker-compose -f docker-compose.yaml -f docker-compose.local.yaml up --build",
```

Comment:
Same as above - why do we need separate docker compose config for local env - we already have the override config?

Reply:
Because the override is being run in sandbox, and we want to keep this only running on local machines.

Reply:
Override is being run in both local and sandbox - there should be no local docker to avoid duplication.